src/components/footer/index.tsx

```diff
@@ -60,6 +62,9 @@ function Footer({
 	openConsentManager,
 	className,
 }: FooterProps): React.ReactElement {
+	const { pathname } = useRouter()
+	const shouldRenderThemeSwitcher = pathname !== '/' && pathname !== 'sign-up'
```

Comment:
Again, will refactor this in a follow up PR https://app.asana.com/0/1203652232454021/1204323512217789